pages/404.tsx

```diff
@@ -0,0 +1,14 @@
+import {ErrorPages} from 'src/components/ErrorPages';
+
+export default function Custom404() {
+
+    const errorNum = 404
+    const errorName = 'Page Not Found'
+    const errorSlogan = "Opps! Sorry, the page you were looking for does not exist."
```

Comment:
This is probably another reason to add ESLint or Prettier to the project but I usually like to enforce `;` at the end of each line. It's not really necessary but just for consistency.

Reply:
I don't think you need to change it now though, when we set up one of those it will take care of it.